src/media/css/detail/reviews.styl

```diff
@@ -28,9 +28,6 @@
 }
 
 @media $base-tablet {
-    .app-reviews h3 {
-        margin-bottom: 20px;
-    }
     .app-reviews .detail-right-side {
```

Comment:
There's an old idea in CSS that you don't name colours, sides etc in case the container get altered at a later date. No biggie though just a nitpick.